apps/dashboard/app/javascript/packs/batch_connect.js

```diff
@@ -116,51 +111,76 @@ function snakeCaseWords(str) {
 }
 
 /**
- *
+ * Get the data attributes for an element similar to jquery .data()
+ * Returns a dict of all keys that start with data- after they have been mountainCased and remove data- prefix
+ * Values of dict are values of attributes starting with data-
+ * 
+ * @param {*} element
+ */
+function getData(element) {
+  let data = {};
+  [...element.attributes].filter(x => x.name.startsWith('data-')).forEach(attr => {
+    data[mountainCaseWords(attr.name.replace('data-', ''))] = attr.value;
+  });
+  return data;
+}
```

Comment:
IDK if we need this - html elements have a dataset api.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset